docs/application-insights.md

```diff
@@ -0,0 +1,14 @@
+- [Application Insights](#application-insights)
+
+## Application Insights
```

Comment:
Every markdown file should have a heading line, this would a good candidate for that heading